### ISSUE TITLE
Visual Accessibility issues on Model properties

### ISSUE THREAD

**Issue by maryscar** [swagger-api/swagger-ui#4539]
## Q&A (please complete the following information)
 - OS: Windows
 - Browser: Chrome
 - Version: 66.0.3359.139 (Official Build) (64-bit)
 - Method of installation: npm
 - Swagger-UI version: 3.14.1
 - Swagger/OpenAPI version: Latest.

Expected Behavior

1 ) Property Style fails color contrast ratio thresholds for accessibility.
Current Behavior

1 )
image
![image](https://user-images.githubusercontent.com/34582625/39893275-1cdb699e-5458-11e8-8203-168dfcba0fdf.png)

Possible Solution

1 ) Increase the weight to the property format.
From #999 to #666

Context

Accessibility testing was completed using internal tools as well as Google Accessibility Extension (https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb) and aXe Accessibility Engine (https://www.deque.com/products/axe/). These failures were the most consistent and effect users with visual accessibility issues. 

Will submit a pull request with fix to object-model.jsx
